snooty.toml

```diff
@@ -3,18 +3,21 @@ title = "MongoDB Analyzer"
 
 [constants]
 product = "MongoDB Analyzer"
-product-home-page = "TODO: Insert homepage link when available"
 product-source-repo = "https://github.com/mongodb/mongo-csharp-analyzer"
-product-nuget-link = "TODO:Link to NuGet Package Page for Product"
+# The home page is the project's Github repo for the time being. This may change
+# to a DevRel article or product page such as
+# https://www.mongodb.com/kafka-connector if someone writes such a page in the future.
+product-home-page = "{+product-source-repo+}"
+product-nuget-link = "https://www.nuget.org/packages/MongoDB.Analyzer"
 product-package-name = "MongoDB.Analyzer"
 product-version = "1.0"
 product-version-full = "{+product-version+}.0"
 product-settings-file-constant = "mongodb.analyzer.json"
-product-jira = "TODO: Link to project people should submit issues to"
+product-jira = "https://jira.mongodb.org/browse/VS"
```

Comment:
I can't see where this link is displayed but other than that, LGTM

Reply:
Currently this link is only used in the FAQ entry for JetBrains Rider. You are correct that is too hard to find.

I've[ created a ticket](https://jira.mongodb.org/browse/DOCSP-20240) for adding an "Issues and Help" page so that the project's JIRA can be located from the left hand TOC of the page.